src/app/zap-templates/zcl/data-model/chip/basic-information-cluster.xml

```diff
@@ -14,17 +14,24 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
-<configurator>
+<!--
+XML generated by Alchemy; DO NOT EDIT.
+Source: src/service_device_management/BasicInformationCluster.adoc
+Parameters: 
+Git: 0.7-summer-2025-588-g5f6649158
+-->
+<configurator xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../zcl.xsd">
   <domain name="CHIP"/>
-
   <struct name="CapabilityMinimaStruct">
     <cluster code="0x0028"/>
-    <item fieldId="0" name="CaseSessionsPerFabric" type="int16u" isNullable="false" writable="false"/>
-    <item fieldId="1" name="SubscriptionsPerFabric" type="int16u" isNullable="false" writable="false"/>
+    <cluster code="0x0039"/>
```

Comment:
Seems like this comment: https://github.com/project-chip/connectedhomeip/pull/38137#discussion_r2022097867, is still present in the latest generated XML.

@hasty, any thoughts on this?